kiwi/bootloader/config/grub2.py

```diff
@@ -1441,15 +1441,18 @@ def _fix_grub_loader_entries_boot_cmdline(self):
             for menu_entry_file in glob.iglob(loader_entries_pattern):
                 with open(menu_entry_file) as grub_menu_entry_file:
                     menu_entry = grub_menu_entry_file.read()
+                    # Made append only
                     menu_entry = re.sub(
                         r'options (.*)',
-                        r'options {0}'.format(self.cmdline),
+                        r'options \1 {0}'.format(self.cmdline),
```

Comment:
This seems to be a generic change not ostree specific and it changes the behavior as we intentionally wanted to replace the complete cmdline with this _fix method. I'm so happy when we can get rid of this _fix stuff

From an implementation point of view I think the different expression should be only applied in case of  a ```boot/ostree``` system. As it seems we have this differentiation at some places it might be a good idea to provide a method in the ```defaults.py``` that offers something like ```def is_ostree() -> bool:```

Reply:
An option might be to add a `kernelcmdlineappend` option that will only append to the command line instead of replacing it entirely. This would not change the behavior of the existing one and would let us use it for the ostree case.

Reply:
Done